bin/image

```diff
@@ -73,6 +83,9 @@ case $2 in
     last4minors)
         versions=$(last4minors)
         ;;
+    new_versions)
```

Comment:
please add a comment here as well, what is the expected input and output?

Reply:
done